src/generic/Wysiwyg.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect, Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { getConfig } from '@edx/frontend-platform';
+import {
+  prepareEditorRef,
+  TinyMceWidget,
+} from '@edx/frontend-lib-content-components';
+
+const store = createStore(() => ({}));
+
+export const SUPPORTED_TEXT_EDITORS = {
+  text: 'text',
+  expandable: 'expandable',
+};
+
+const mapStateToProps = () => ({
+  assets: {},
+  lmsEndpointUrl: getConfig().LMS_BASE_URL,
+  studioEndpointUrl: getConfig().STUDIO_BASE_URL,
+  isLibrary: {},
+  onEditorChange: () => ({}),
+});
+
+const Editor = connect(mapStateToProps)(TinyMceWidget);
+
+export const Wysiwyg = ({ initialValue, editorType, onChange }) => {
+  const defaultEmptyTextValue = '<p>&nbsp;</p>';
+  const { editorRef, refReady, setEditorRef } = prepareEditorRef();
+
+  const isEquivalentCodeQuotes = (first, second) => {
+    // Utils allows to compare code with single quotes and double quotes
+    const normalizeQuotes = (section) => section.replace(/'/g, '"');
+    return normalizeQuotes(first) === normalizeQuotes(second);
```

Comment:
What will be the behavior if the user enters `""Test""` (quotes inside other quotes)?